README.md

```diff
@@ -35,115 +35,75 @@ Terminology can be confusing , e.g. Cartographic label != ML label, please ask i
-
-13:00-16:00
-* Exercise 8: CNN based image segmentation with keras
-* Lecture 8: GIS software supporting machine learning for spatial data. 
-* Wrap-up and where to go from here
+## Schedule
+https://ssl.eventilla.com/event/VDK2b
 
 ## Course computing enviroment
 
-Almost all exercises will be done in "Jupyter notebooks for courses" in the [Puhti webinterface](https://www.puhti.csc.fi) .
+First 2 days the exercises are done in "Jupyter notebooks for courses" in the [Puhti webinterface](https://www.puhti.csc.fi) . 
 Puhti is one of CSCs supercomputers providing researchers in Finland with High Performance Computing resources.
 Course participants will be provided with a temporary account (training_xxx); it is not possible to use your own CSC account. 
 
-* Puhti webinterface & accounts
-    1. connect to https://www.puhti.csc.fi
-    2. enter provided username (training_xxx) and password 
-    3. find and click "Jupyter for courses" on dashboard
-    4. choose Project "project_2002044" first!
-    5. then choose Module "GeoML22"
-    6. and set workingdirectory to "/scratch/project_2002044"
-    7. click launch and wait until granted resources
-    8. then click "Connect to Jupyter" 
-
-We will use the Jupyter for courses environment on Day 1 and 2, on day 3 we will use Jupyter and the batch job submission system "the normal way" (the same way that you can also use Puhti after the course).
-
-... to be continued within intro.ipynb
+### Puhti webinterface
+* Open https://www.puhti.csc.fi
+* Enter provided username (training_xxx) and password 
+    
+### Jupyter for courses
+* Click "Jupyter for courses" on dashboard
+* Select:
+   * Project: project_2002044
+   * Module: GeoML22
+   * Reservation: geoml
+   * Working directory: "/scratch/project_2002044"
+* Click launch and wait until granted resources
+* Click "Connect to Jupyter" 
+* ... to be continued within intro.ipynb
+
+### QGIS
+* Click "Desktop" on dashboard
+* Select:
+   * Project: project_2002044
+   * Partition: small
+   * Reservation: geoml
+   * Number of CPU cores: 1
+   * Memory (GB): 8 Gb
+   * Local disk: 0
+   * Time: 4:00:00 (or adjust to reasonable)
+   * Desktop: single application
+   * App: QGIS
+* Click launch and wait until granted resources
+* Click "Launch Desktop" 
 
 ## Content of this repository
 
-This repository contains all Jupyter notebooks used in the course.
-
-**01_clustering**
-
-**02_vector_data_preparation**
-
-* Preparation of Paavo zip code dataset for machine learning
-* Input:
-    * Paavo zip code dataset
-    * counties dataset
-* Output:
-    * scaled and unscaled train, test validation datasets and labels
+This repository contains all Jupyter Notebooks and other code used in the course. Data is not inlcuded here, data download links are provided in data preparations Notebooks. Each exercise has its own folder:
 
-**03_raster_data_preparation**
+* [01_clustering](01_clustering)
+* [02_vector_data_preparation](02_vector_data_preparation)
```

Comment:
```suggestion
* [02_vector_data_preparation](02_vector_data_preparation)
* Preparation of Paavo zip code dataset for machine learning
    * Input:
        * Paavo zip code dataset
        * Regions dataset
    * Output:
        * Scaled and unscaled train, test validation datasets and labels
```

Reply:
I would not add these here, these should be clearly in the beginning of each Notebook.
I do not like to have same things in many places -> sooner or later things will be not in sync any more..